client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/CustomEventLogListener.java

```diff
@@ -250,6 +287,7 @@ public CustomEvent convert(Integer key, String previousValue, Metadata previousM
       static class FilterConverter extends AbstractCacheEventFilterConverter<Integer, String, CustomEvent>
          implements Serializable {
          private final Object[] params;
+         private final CallbackCounter counter = new NumericCallbackCounter();
```

Comment:
Actually the failure cause seems to be different, but I would still use AtomicInteger for simplicity.